python/ray/train/v2/_internal/constants.py

```diff
@@ -41,34 +44,42 @@
 REPORT_BARRIER_WARN_INTERVAL_S_ENV_VAR = "RAY_TRAIN_REPORT_BARRIER_WARN_INTERVAL_S"
 DEFAULT_REPORT_BARRIER_WARN_INTERVAL_S: float = 60
 
-# The environment variable to enable the Ray Train Metrics.
-METRICS_ENABLED_ENV_VAR = "RAY_TRAIN_METRICS_ENABLED"
-
 # Environment variable to enable the print function patching.
 ENABLE_PRINT_PATCH_ENV_VAR = "RAY_TRAIN_ENABLE_PRINT_PATCH"
 DEFAULT_ENABLE_PRINT_PATCH = "1"
 
-# Whether or not to run the controller as an actor.
-RUN_CONTROLLER_AS_ACTOR_ENV_VAR = "RAY_TRAIN_RUN_CONTROLLER_AS_ACTOR"
-DEFAULT_RUN_CONTROLLER_AS_ACTOR = "1"
-
 # V2 feature flag.
 V2_ENABLED_ENV_VAR = "RAY_TRAIN_V2_ENABLED"
 
-
-def is_v2_enabled() -> bool:
-    return env_bool(V2_ENABLED_ENV_VAR, False)
-
-
+# Environment variables to propagate from the driver to the controller,
+# and then from the controller to the workers.
 ENV_VARS_TO_PROPAGATE = {
     V2_ENABLED_ENV_VAR,
     HEALTH_CHECK_INTERVAL_S_ENV_VAR,
     WORKER_HEALTH_CHECK_TIMEOUT_S_ENV_VAR,
     WORKER_GROUP_START_TIMEOUT_S_ENV_VAR,
+    REPORT_BARRIER_TIMEOUT_S_ENV_VAR,
+    REPORT_BARRIER_WARN_INTERVAL_S_ENV_VAR,
```

Comment:
driveby fix -- these env vars are used by the `WorkerGroup`, which lives in the controller process